vignettes/orthophotomap.Rmd.orig

```diff
@@ -47,7 +47,7 @@ Values close to 0 and below are related to water, bare soil surfaces or building
 ## Analysis
 
 ```{r message=FALSE}
-# attach packages
+# load packages
```

Comment:
Same here.

Reply:
Plus in **topodb.Rmd**. I will change it.

Reply:
👍🏻

Reply:
Or is it should be formally load and attach? From `library()` docs:

> library(package) and require(package) both load the namespace of the package with name package and attach it on the search list.

Reply:
Good point. Probably both together are the most correct.

Reply:
Let's stay with your proposal to avoid confiusion.